desktop.ini

```diff
@@ -0,0 +1,4 @@
+[ViewState]
```

Comment:
that file is needed?

Reply:
The file is not needed, it has been deleted.